service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/judicial/NewHearingJudgeEventHandler.java

```diff
@@ -51,14 +51,18 @@ public void handleNewHearingJudge(final NewHearingJudgeEvent event) {
         ZonedDateTime possibleEnd = nextHearing.map(hearing -> hearing.getStartDate().atZone(ZoneId.systemDefault()))
             .orElse(null);
 
+        // if this is the first hearing, use now as the start date/time, else use the start of the hearing
```

Comment:
Hi Dan, if you broke down the logic a bit you wouldn't need the comment, e.g.:

```boolean isFirstHearing = event.getCaseData().getAllNonCancelledHearings().size() > 1 ? false : true;

ZonedDateTime startDate = isFirstHearing
            ? ZonedDateTime.now()
            : event.getHearing().getStartDate().atZone(ZoneId.systemDefault());```

Reply:
added!